game-core/src/main/java/games/strategy/triplea/delegate/battle/steps/fire/FiringGroupSplitterBombard.java

```diff
@@ -0,0 +1,37 @@
+package games.strategy.triplea.delegate.battle.steps.fire;
+
+import static games.strategy.triplea.delegate.battle.BattleState.Side.OFFENSE;
+import static games.strategy.triplea.delegate.battle.BattleState.UnitBattleFilter.ALIVE;
+import static games.strategy.triplea.delegate.battle.BattleStepStrings.NAVAL_BOMBARD;
+
+import games.strategy.engine.data.Unit;
+import games.strategy.triplea.delegate.Matches;
+import games.strategy.triplea.delegate.battle.BattleState;
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Function;
+import lombok.Value;
+import org.triplea.java.collections.CollectionUtils;
+
+/**
+ * Create naval bombardment firing groups
+ *
+ * <p>The firing groups are separated by isSuicideOnHit
+ */
+@Value(staticConstructor = "of")
+public class FiringGroupSplitterBombard implements Function<BattleState, List<FiringGroup>> {
+
+  @Override
+  public List<FiringGroup> apply(final BattleState battleState) {
+    final Collection<Unit> canFire = battleState.getBombardingUnits();
```

Comment:
nit, would 'bombarindUnits' be a more descriptive name here? Perhaps even consider inlining it to avoid the intermediate variable?